sku.0/sys.server/compiled/game/script/item/goggles/reset.java

```diff
@@ -24,6 +24,13 @@ public reset()
     };
     public int createNewGoggles(obj_id self, obj_id target, String params, float defaultTime) throws InterruptedException
     {
+    
+        String config = getConfigSetting("GameServer", "seGogglesReward");

```

Comment:
This has nothing to do with food assembly, lol

Reply:
I added this fix to my repo and it added it to the pull request. Oops.